src/Swift.Runtime/src/Swift/FoundationDatabase.xml

```diff
@@ -53,16 +53,22 @@
             <typedeclaration kind="struct" name="UnsafeMutableRawPointer" module="Swift" mangledName="sSv" frozen="true" blittable ="true"/>
         </entity>
         <entity managedNameSpace="Swift" managedTypeName="UnsafeBufferPointer">
-            <typedeclaration kind="struct" name="UnsafeBufferPointer`1" module="Swift" mangledName="sSR" frozen="false" blittable ="false"/>
+            <typedeclaration kind="struct" name="UnsafeBufferPointer" module="Swift" mangledName="sSR" frozen="false" blittable ="false"/>
         </entity>
         <entity managedNameSpace="Swift" managedTypeName="UnsafeMutableBufferPointer">
-            <typedeclaration kind="struct" name="UnsafeMutableBufferPointer`1" module="Swift" mangledName="sSr" frozen="false" blittable ="false"/>
+            <typedeclaration kind="struct" name="UnsafeMutableBufferPointer" module="Swift" mangledName="sSr" frozen="false" blittable ="false"/>
         </entity>
         <entity managedNameSpace="Swift" managedTypeName="UnsafeRawBufferPointer">
             <typedeclaration kind="struct" name="UnsafeRawBufferPointer" module="Swift" mangledName="sSW" frozen="true" blittable ="true"/>
         </entity>
         <entity managedNameSpace="Swift" managedTypeName="UnsafeMutableRawBufferPointer">
             <typedeclaration kind="struct" name="UnsafeMutableRawBufferPointer" module="Swift" mangledName="sSw" frozen="true" blittable ="true"/>
         </entity>
+        <entity managedNameSpace="Swift" managedTypeName="UnsafeMutablePointer&lt;System.Byte&gt;">
```

Comment:
Please add a tracking issue so it doesn't get lost.

Reply:
https://github.com/dotnet/runtimelab/issues/2979